app/src/main/java/com/kickstarter/libs/utils/SharedFunctions.kt

```diff
@@ -61,3 +65,20 @@ fun checkoutProperties(
         .shippingAmount(shippingAmount)
         .build()
 }
+
+fun <T : Relay> encodeRelayId(relay: T): String {
+    val classSimpleName = relay.javaClass.simpleName.replaceFirst("AutoParcel_", "")
+    val id = relay.id()
+    return Base64Utils.encodeUrlSafe(("$classSimpleName-$id").toByteArray(Charset.defaultCharset()))
+}
+
+fun decodeRelayId(encodedRelayId: String?): Long? {
```

Comment:
Same here, `decodeRelayId` can be an extension to the `String` class in case you need to extract this method